.lando.yml

```diff
@@ -0,0 +1,25 @@
+name: starshot
+recipe: drupal10
+config:
+  webroot: web
+services:
+  appserver:
+    overrides:
+      environment:
+        DRUSH_OPTIONS_URI: "https://starshot.lndo.site"
+tooling:
+  install:
+    service: appserver
+    cmd:
+      - composer drupal:install
+  login:
+    service: appserver
+    cmd:
+      - echo "🚀 Board the spaceship! Click on the link below to login 👇"
+      - |
+        echo "
+        # Tell Drush about your website.
+        options:
+          uri: 'https://starshot.lndo.site'
+        " > drush/drush.yml
```

Comment:
I'm not sure we need to change `drush.yml`, since we're injecting `DRUSH_OPTIONS_URI` into the container (I think).

Reply:
Another thing I realize this is missing -- in the DDEV version, a browser window automatically opens up. Can we get the same thing happening with Lando?

Reply:
I needed to change `drush.yml` because this file takes over the variables injected :face_with_diagonal_mouth: 

AFAIK there is no command in Lando to `launch` the website, unfortunately.

I've updated the `.lando.yml` trying to keep it simple.

Reply:
> AFAIK there is no command in Lando to launch the website, unfortunately.

Well, it's not that we need _Lando_ to have a command for this -- but for parity with our DDEV script, we need be able to call `open` _on the host machine_ -- that is, from _outside_ the appserver container. DDEV has a way to call commands on the host machine (putting the script in `.ddev/commands/host`). Does Lando have that capability?

Reply:
> I needed to change drush.yml because this file takes over the variables injected 🫤

🤔 Hmmm...my understanding, from discussion with the Drush maintainers, is that the environment variables should supersede what's in `drush.yml`. There was a bug around this, but I think they merged it? Maybe it just hasn't made it into a release yet.

Reply:
Nope, unfortunately this is not possible AFAIK (see [this thread](https://github.com/lando/lando/issues/847#issuecomment-792800913))